### ISSUE TITLE
Enable streaming for restricted bitstreams

### ISSUE THREAD

**Issue by Marwa1988-S** [DSpace/dspace-angular#2223]
**Describe the bug**
 Media-viewer in item page works properly only if the bitstream is public, the image/video is displayed correctly only if this bitstream is accessible with 'Anonymous' group.
it should support the use case of providing streaming to the group it's access restricted to.
(test done in google chrome and firefox - Dspace 7.5).

**To Reproduce**
1. Login as Administrator 
2. Navigate to an item page, add image or video
3. Edit this item
4. Edit bitstream Policies (for bundle Original)
5. Change the eperson or group that will be granted the permission to Administrator then save
6. Back to item page, the media viewer doesn't display the file

**Expected behavior**
Streaming should work when the logged in user has permission to see/download the file.

**An Example**
https://demo7.dspace.org/items/0eda4c93-0915-4ad9-b51c-dd3f8a30a930

**Comment by alexandrevryghem** [comment]
This is a duplicate of #1887

**Comment by tdonohue** [comment]
Closing as duplicate of #1887.  See that ticket for more details. It is waiting on a volunteer.
